packages/ui/src/components/icons/Docs.tsx

```diff
@@ -0,0 +1,12 @@
+export function Docs() {
```

Comment:
Can we use the [help icon](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Ahelp_outline%3A) instead?